pr_agent/algo/ai_handler.py

```diff
@@ -68,22 +71,29 @@ def deployment_id(self):
         """
         return get_settings().get("OPENAI.DEPLOYMENT_ID", None)
 
-    @retry(exceptions=(APIError, Timeout, TryAgain, AttributeError, RateLimitError),
-           tries=OPENAI_RETRIES, delay=2, backoff=2, jitter=(1, 3))
-    async def chat_completion(self, model: str, system: str, user: str, temperature: float = 0.2):
+    @retry(
+        exceptions=(APIError, TryAgain, AttributeError, RateLimitError),
+        tries=OPENAI_RETRIES,
+        delay=2,
+        backoff=2,
+        jitter=(1, 3),
+    )
```

Comment:
**Suggestion:** It's a good practice to avoid using magic numbers directly in the code. The numbers used in the `retry` decorator in the `chat_completion` method (like `2`, `1`, `3`) could be replaced with well-named constants to improve code readability and maintainability. [best practice]
```suggestion
    DELAY = 2
    BACKOFF = 2
    JITTER = (1, 3)

    @retry(
        exceptions=(APIError, TryAgain, AttributeError, RateLimitError),
        tries=OPENAI_RETRIES,
        delay=DELAY,
        backoff=BACKOFF,
        jitter=JITTER,
    )
```